contracts/integrations/test/mainnet_configs_test.ts

```diff
@@ -0,0 +1,200 @@
+        });
+    });
+    describe('ERC20BridgeProxy', () => {
+        const erc20BridgeProxy = new ERC20ProxyContract(contractAddresses.erc20BridgeProxy, env.provider);
+        it('should be owned by the ZeroExGovernor ', async () => {
+            const owner = await erc20BridgeProxy.owner().callAsync();
+            expect(owner).to.eq(contractAddresses.zeroExGovernor);
+        });
+        it('should have the correct authorized addresses', async () => {
+            const authorizedAddresses = await erc20BridgeProxy.getAuthorizedAddresses().callAsync();
+            expect(authorizedAddresses.length).to.eq(2);
+            expect(authorizedAddresses.includes(contractAddresses.exchange), 'Exchange');
+            expect(authorizedAddresses.includes(contractAddresses.multiAssetProxy), 'MultiAssetProxy');
+        });
+    });
+    describe('MultiAssetProxy', () => {
+        const multiAssetProxy = new MultiAssetProxyContract(contractAddresses.multiAssetProxy, env.provider);
+        it('should be owned by the ZeroExGovernor ', async () => {
+            const owner = await multiAssetProxy.owner().callAsync();
+            expect(owner).to.eq(contractAddresses.zeroExGovernor);
+        });
+        it('should have the correct authorized addresses', async () => {
+            const authorizedAddresses = await multiAssetProxy.getAuthorizedAddresses().callAsync();
+            expect(authorizedAddresses.length).to.eq(2);
+            expect(authorizedAddresses.includes(contractAddresses.exchangeV2), 'ExchangeV2');
+            expect(authorizedAddresses.includes(contractAddresses.exchange), 'Exchange');
+        });
+        it('ERC20Proxy should be registered', async () => {
+            const erc20Proxy = await multiAssetProxy.getAssetProxy(AssetProxyId.ERC20).callAsync();
+            expect(erc20Proxy).to.eq(contractAddresses.erc20Proxy);
+        });
+        it('ERC721Proxy should be registered', async () => {
+            const erc721Proxy = await multiAssetProxy.getAssetProxy(AssetProxyId.ERC721).callAsync();
+            expect(erc721Proxy).to.eq(contractAddresses.erc721Proxy);
+        });
+        it('ERC1155Proxy should be registered', async () => {
+            const erc1155Proxy = await multiAssetProxy.getAssetProxy(AssetProxyId.ERC1155).callAsync();
+            expect(erc1155Proxy).to.eq(contractAddresses.erc1155Proxy);
+        });
+        it('ERC20BridgeProxy should be registered', async () => {
+            const erc20BridgeProxy = await multiAssetProxy.getAssetProxy(AssetProxyId.ERC20Bridge).callAsync();
+            expect(erc20BridgeProxy).to.eq(contractAddresses.erc20BridgeProxy);
+        });
+        it('StaticCallProxy should be registered', async () => {
+            const staticCallProxy = await multiAssetProxy.getAssetProxy(AssetProxyId.StaticCall).callAsync();
+            expect(staticCallProxy).to.eq(contractAddresses.staticCallProxy);
+        });
+    });
+    describe('StakingProxy', () => {
```

Comment:
Could be good to test the storage layout here as well.

Reply:
👍 Added a task to do this in a separate PR